README.md

```diff
@@ -3,6 +3,7 @@ Portfolio:
 MileStone 1:
 >Portfolio for Mobile Version.
 >Its responsive
+>The units like rem, em, vh ,vw are used.
 
```

Comment:
- [ ] Kindly make sure that the live demo link works and redirects the user to the demo page. Currently, it's not working.

Reply:
It is working Bother i just checked @ShadyShawkat

Reply:
The link in the README goes to this link (https://github.com/caasperr/Portfolio/blob/Deploy-Website/caasperr.github.io) which is not the correct link.

Reply:
i am sorry i pushed the actual link to main branch now can you check again please? @ShadyShawkat

Reply:
It is working fine now @caasperr 👏🏻. Kindly resubmit your project.

Reply:
I did that Thankyou soo much @ShadyShawkat